PROTO_tests/tests/client_test.py

```diff
@@ -0,0 +1,111 @@
+import arkouda as ak
+import pytest
+import os
+
+from server_util.test.server_test_util import start_arkouda_server
+
+
+class TestClient:
+    def test_client_connected(self):
+        """
+        Tests the following methods:
+        ak.client.connected()
+        ak.client.disconnect()
+        ak.client.connect()
+
+        :return: None
+        :raise: AssertionError if an assert* method returns incorrect value or
+                if there is a error in connecting or disconnecting from  the
+                Arkouda server
+        """
+        assert ak.client.connected
+        try:
+            ak.disconnect()
+        except Exception as e:
+            raise AssertionError(e)
+
+        assert not ak.client.connected
+        try:
+            ak.connect(server=pytest.server, port=pytest.port)
+        except Exception as e:
+            raise AssertionError(e)
+
+        assert ak.client.connected
+
+    def test_disconnect_on_disconnected_client(self):
+        """
+        Tests the ak.disconnect() method invoked on a client that is already
+        disconnect to ensure there is no error
+        """
+        ak.disconnect()
+        assert not ak.client.connected
+        ak.disconnect()
+        ak.connect(server=pytest.server, port=pytest.port)
+
+    def test_shutdown(self):
+        """
+        Tests the ak.shutdown() method
+        """
+        ak.shutdown()
+        start_arkouda_server(numlocales=pytest.nl)
```

Comment:
i just want to double check, have we ran `make test` with multi-locale to verify it works. 

Running it locally the only way i can get any of the tests to work in multi-locale is with `ARKOUDA_RUNNING_MODE=CLIENT` and then i have to comment out `test_shutdown`. I might be doing something wrong

Reply:
it's worth noting that i think i have the same issue with the current `make test`

Reply:
That is odd. I specifically tested this because I thought it could be an issue. I just reran it and verified that it was successful for me. Output is below. When you get in, can you show me what is happening for you?

```
(ak-dev) ➜  arkouda git:(2538_client_test_upd) ✗ python3 -m pytest tests/tests/client_test.py 
========================================================== test session starts ==========================================================
platform darwin -- Python 3.10.10, pytest-7.3.1, pluggy-1.0.0
benchmark: 4.0.0 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /Users/ethandebandi/Documents/git/arkouda
configfile: pytest.ini
plugins: env-0.8.1, typeguard-2.10.0, json-report-1.5.0, metadata-3.0.0, benchmark-4.0.0
collected 9 items                                                                                                                       

tests/tests/client_test.py .........                                                                                              [100%]

=========================================================== warnings summary ============================================================
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:121
  /Users/ethandebandi/anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:121: DeprecationWarning: pkg_resources is deprecated as an API
    warnings.warn("pkg_resources is deprecated as an API", DeprecationWarning)

../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
  /Users/ethandebandi/anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('mpl_toolkits')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
../../../anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870
  /Users/ethandebandi/anaconda3/envs/ak-dev/lib/python3.10/site-packages/pkg_resources/__init__.py:2870: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('sphinxcontrib')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
===================================================== 9 passed, 8 warnings in 2.93s =====================================================
(ak-dev) ➜  arkouda git:(2538_client_test_upd) ✗ 
```

Reply:
I also wanted to note that I ran with 2, 3, 4, and 5 locales for this. All functioned as expected for me.

Reply:
I AM DUMB! @Ethan-DeBandi99 and looked at this and it was very silly mistake (i was forgetting to enable gasnet in the other terminal 🤦 )